bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/Configuration.kt

```diff
@@ -213,6 +214,8 @@ data class CaptureConfiguration(
      * Enable/disable the skonto feature.
      */
     val skontoEnabled: Boolean = true,
+
+    val userCommentGiniBankVersion: String = "GiniBankVer",
```

Comment:
[Request] This can be a simple constant. No need for clients to be able to configure this.